templates/kotsadm-service.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if not .Values.kurlProxy.enabled }}
```

Comment:
could this break existing integrations that might rely on this `admin-console` service being in the cluster?

Reply:
Nothing besides embedded-cluster uses this helm chart, and we don't rely on that service existing

Reply:
I'll add a `service.enabled` field which to allow embedded-cluster to explicitly disable the service, and also remove this dependency.

Reply:
ah, this makes sense. i think i was mixing this service up with the internal kotsadm service.